vizro-mcp/README.md

```diff
@@ -4,31 +4,30 @@
 
 # Vizro-MCP
 
-Vizro-MCP is a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/) server, which works alongside a LLM to help you create Vizro dashboards and charts.
+Vizro-MCP is a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/) server, which works alongside an LLM to help you create Vizro dashboards and charts.
 
 <img src="docs/assets/images/vizro-mcp.gif" width="600" alt="Vizro-MCP Demo">
 
+To find out more, consult the [Vizro-MCP documentation](https://vizro.readthedocs.io/projects/vizro-mcp/).
+
 ## Set up Vizro-MCP
 
 Vizro-MCP is best used with Claude Desktop, Cursor or VS Code. However, it can be used with most LLM products that enable configuration of MCP server usage.
 
 > 💡 Tip: For best performance, we recommend using the `claude-4-sonnet` model, or another high-performing model of your choice. Using the often offered `auto` setting may lead to inconsistent or unexpected results.
 
-Our documentation offers separate, detailed steps for [Claude Desktop](./docs/pages/guides/set-up-vizro-mcp-with-claude.md), [Cursor](./docs/pages/guides/set-up-vizro-mcp-with-cursor.md) and [VS Code](./docs/pages/guides/set-up-vizro-mcp-with-vscode.md).
+Our documentation offers separate, detailed steps for [Claude Desktop](https://vizro.readthedocs.io/projects/vizro-mcp/en/latest/pages/guides/set-up-vizro-mcp-with-claude/), [Cursor](https://vizro.readthedocs.io/projects/vizro-mcp/en/latest/pages/guides/set-up-vizro-mcp-with-cursor/) and [VS Code](https://vizro.readthedocs.io/projects/vizro-mcp/en/latest/pages/guides/set-up-vizro-mcp-with-vscode/).
```

Comment:
These have to point to `en/latest` as they're not pointing into the root of the docs so need a version/language.